client/verta/verta/data_types/_discrete_histogram.py

```diff
@@ -59,3 +63,27 @@ def _from_dict_inner(cls, d):
             buckets=data["buckets"],
             data=data["data"],
         )
+
+    def dist(self, other):
+        if not isinstance(other, type(self)):
+            raise TypeError(
+                "`other` must be type {}, not {}".format(type(self), type(other))
+            )
+
+        # TODO: assuming labels are consistent
```

Comment:
I'd like to get this right within the scope of this PR because I think it will be more work to circle back and fix it than to do it now. I can jump in and provide the implementation if you would like.

Reply:
I believe I've got this.